aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -10,7 +10,7 @@
 from fsspec.spec import AbstractFileSystem
 from ome_types import from_xml
 from ome_types.model.ome import OME
-from tifffile import TiffFile, TiffFileError, TiffTag
+from tifffile.tifffile import TiffFile, TiffFileError, TiffTag, TiffTags
```

Comment:
I see that TiffTag and TiffTags both get used. On first glance I thought it was a typo. What's the difference if I might ask?

Reply:
TiffTag represents a single TIFF tag structure.
TiffTags is a multidict like interface to TiffTag instances belonging to a TiffPage.
https://github.com/cgohlke/tifffile/blob/master/tifffile/tifffile.py#L7553-L7568